news/_posts/2020-05-07-productboard-portal-launch.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title:  "A new place to contribute to the PrestaShop project and share your feedback on the next features"
+subtitle: "Discover PrestaShop's new tool to collect feedback and better prioritize what to build next!"
+date:   2020-05-07 10:00:00
+authors: [ SamuelPires ]
+image:   /assets/images/theme/meta-logo-build.png
+icon: icon-lab
+tags:
+ - contribute
+ - design
+ - feature
+ - news
+ - process
+ - product
+---
+
+As announced [here](https://build.prestashop.com/news/product-features-methodology-at-prestashop), the Core product team recently adopted a new working tool to gather feedback and prioritize features: [Productboard](https://www.productboard.com). No more teasing, it is now time to have a look at how it will look and how we will all be able to handle it!
+
+Productboard makes two things possible for the team:
+
+- On the one hand, having _a dedicated internal interface_ to centralize all feedback, group them into topics, define features, and prioritize them.
+- On the other hand, having _an open public portal_ to explicitly present those features and collect feedback about them. **And that is what this article is about.**
+
+So the team has created this portal called **PrestaShop Feedback** so that anyone can comment and vote for the features on which feedback is needed. Whether you are a merchant, a developer, an agency, a seller on the [Addons Marketplace](http://addons.prestashop.com/), etc. your opinions and suggestions are expected on the [feedback.prestashop.com](https://feedback.prestashop.com) page.
+
+The portal, available in the 'PrestaShop Core' tab, is divided into several sections. The goal of this article is to explain how to use this interface to share your feedback on upcoming features.
```

Comment:
do we also talk about the prestashop checkout portal ?

Reply:
maybe not "upcoming" as this might sound like a roadmap. Maybe just "features the team is working on"

Reply:
```suggestion
The portal, available in the 'PrestaShop Core' tab, is divided into several sections. The goal of this article is to explain how to use this interface to share your feedback on the features the team is working on.
```